tool/tctl/common/collection.go

```diff
@@ -715,14 +723,18 @@ func (c *windowsDesktopCollection) resources() (r []types.Resource) {
 }
 
 func (c *windowsDesktopCollection) writeText(w io.Writer) error {
-	t := asciitable.MakeTable([]string{"UUID", "Address"})
+	t := asciitable.MakeTable([]string{"Host", "Public Address", "AD Domain", "Labels", "Version"})
```

Comment:
I'm not sure I would say "Public Address" here. "Public" may be seen as routable and that's often not going to be true.